src/AppBundle/Controller/PageController.php

```diff
@@ -93,6 +94,19 @@ public function mouvementOrganisationAction()
     }
 
     /**
+     * @Route("/le-mouvement/la-carte", name="page_le_mouvement_la_carte")
+     * @Method("GET")
+     */
+    public function mouvementCarteAction()
+    {
+        return $this->render('page/la-carte.html.twig', [
+            'committees' => $this->getDoctrine()->getRepository(Committee::class)->findBy([
```

Comment:
it's better to have an explicit method in the repository class to fetch approved committees rather than using Doctrine built-in finders.